osu.Game/Screens/Menu/ButtonArea.cs

```diff
@@ -72,6 +72,7 @@ public ButtonSystemState ButtonSystemState
         public Visibility State
         {
             get => state;
+
```

Comment:
inconsistent with the code style, we have the setter in a new line after getter:
```csharp
get => ...
set => ...
```
(it's already obvious i don't know why you changed it)

Reply:
Srry, bad settings of autocorrection. When I would at home, I change it